### PR TITLE
fix: logout

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14.18.1
```
Comment (geka-evk) [inline .nvmrc:1]: Should the version be 18?

**Review by geka-evk — APPROVED** [review]

File: src/Auth/sagas.ts [hunk 2]
```diff
@@ -86,7 +86,16 @@ function* doAuth() {
...
+  const loginProvider: string = yield select(selectors.getLoginProvider);
+  if (loginProvider) {
+    fetch(logoutEndpoint, { headers: { accept: 'application/json' } })
+      .then((response) => response.json())
+      .then((jsonResponse) => {
+        window.location.assign(`${jsonResponse.logout_url}&return_to=${window.location.href}`);
+      });
+  } else {
+    window.location.assign(`${logoutEndpoint}?return_to=${window.location.href}`);
+  }
```
Comment (kalinkrustev) [inline src/Auth/sagas.ts:98]: Small note - this implementation differs from the other portals in the following details:
- Here the 2 step logout process will only happen if `LOGIN_PROVIDER` is set.
- The other portals pass `return_to` to the fetch.

**Review by kalinkrustev — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (vijayg10) [reply .nvmrc:1]: I placed it based on the version in docker image. I think we need to have followup stories to update the node version.

File: src/Auth/sagas.ts (hunk 2 above)
Comment (vijayg10) [reply src/Auth/sagas.ts]: You right. I don't see any reason this implementation be different from others. I refactored to the other portal standard.

**Pull request merged**